bibliopixel/remote/control.py

```diff
@@ -14,89 +13,95 @@
     'valid': True
 }
 
-DEFAULT_SERVER_CONFIG = {
-    'title': 'BiblioPixel Remote',
-    'bgcolor': '#000000',
-    'font_color': '#ffffff',
-    'external_access': False,
-    'port': 5000
+
+OFF_ANIM = {
+    'animation': 'bibliopixel.animation.off.OffAnim',
+    'run': {'threaded': True}
 }
 
+
 BAD_DEFAULT_ERROR = """\
 `{}` is not a valid default!
 It must be one of the configured animation names."""
 
 
-class RemoteControl:
-    def __init__(self, config, animations):
-        # These start with the defaults
-        self.config = dict(DEFAULT_SERVER_CONFIG, **config)
+class RemoteControl(collection.Collection):
+    def __init__(self, layout, animations,
+                 external_access=False, port=5000,
+                 title='BiblioPixel Remote', bgcolor='black',
+                 font_color='white', default=None):
```

Comment:
Line #46. I call it late because I want to get mangling of names and failing on bad names out of the way first. THEN I call super().